geotrek/flatpages/templates/flatpages/flatpage_form.html

```diff
@@ -177,9 +177,13 @@ <h1>{% trans 'Flat page' %}
         $('#save_changes').click(function () {
             $('textarea[id^=id_content_]').each(function () {
                 var gm = $(this).next().data('gridmanager');
-                gm.deinitCanvas();
```

Comment:
You removed `gm.deinitCanvas()`. What was its purpose?

Reply:
No it was not